pkg/apis/v1beta1/nodepool.go

```diff
@@ -137,6 +141,22 @@ const (
 	ConsolidationPolicyWhenUnderutilized ConsolidationPolicy = "WhenUnderutilized"
 )
 
+// DisruptionReason defines valid reasons for disruption budgets.
+// +kubebuilder:validation:Enum={underutilized,expired,empty,drifted}
+type DisruptionReason string
+
+const (
+	DisruptionReasonUnderutilized DisruptionReason = "underutilized"
+	DisruptionReasonExpired       DisruptionReason = "expired"
+	DisruptionReasonEmpty         DisruptionReason = "empty"
+	DisruptionReasonDrifted       DisruptionReason = "drifted"
```

Comment:
These also look close to [Nodeclaim Status Conditions](https://github.com/kubernetes-sigs/karpenter/blob/120c1bc0ae8c20eb46b5edf52327a58429cb992d/pkg/apis/v1beta1/nodeclaim_status.go#L63C1-L66C44) should we be using the same constants?